ibc/main_test.go

```diff
@@ -73,6 +91,17 @@ func prepareEnvironment(
 	err = testPersistenceMod.Start()
 	require.NoError(t, err)
 
+	testConsensusMod := newTestConsensusModule(bus)
+	err = testConsensusMod.Start()
+	require.NoError(t, err)
+
+	// TODO: I want to mock this as using a real p2p module is hard to test
```

Comment:
Can you elaborate on what you mean here? I wasn't able to follow.

Consider using a specific [TODO type](https://github.com/pokt-network/pocket/blob/main/Makefile#L448).

Reply:
This was a personal comment for myself ahah. I think this comment is removed when the P2P mock comes in.